Formula/yugabyte.rb

```diff
@@ -0,0 +1,82 @@
+class Yugabyte < Formula
+  desc "High-performance distributed SQL database Yugabyte DB"
+  homepage "https://yugabyte.com"
+  url "https://downloads.yugabyte.com/yugabyte-2.0.5.2-darwin.tar.gz"
+  sha256 "8d9b939fe10ba4116c569e27b5b177e9249e448a369dc92b42f5f4a41fa5c8b6"
+  depends_on :java => "1.8"
+  def install
```

Comment:
So if we install both yugabyte and yugabyte-cli we will have two copies of the tar.gz on the local machine? Is there any way in brew to share the installed files across these packages? I couldn't find any way to do this but just checking with you in case.